packages/app/src/cli/services/transform.ts

```diff
@@ -0,0 +1,267 @@
+  }
+
+  const transformInfo: TransformInfo = await import(transformPath)
+  const extensions = transformInfo.extensions ?? ['js', 'jsx', 'ts', 'tsx']
+  const includePath = options.include ?? './'
+  const globPattern = getGlobPattern(includePath, extensions)
+  const filePaths = await glob(globPattern, {cwd: cwd()})
+
+  if (filePaths.length === 0) {
+    throw new AbortError(`No files found for ${options.include}`)
+  }
+
+  const transformOptions: {[key: string]: string} = {}
+  for (const option of Object.values(transformInfo.options)) {
+    const message = option.description.replace(/\.$/, '')
+
+    // eslint-disable-next-line no-await-in-loop
+    const input = await renderTextPrompt({
+      message: `${message} ${option.required ? '' : '(optional)'}`,
+      allowEmpty: !option.required,
+    })
+
+    transformOptions[option.name] = input
+  }
+
+  try {
+    const res = await jscodeshift.run(transformPath, filePaths, {
+      babel: true,
+      silent: true,
+      stdin: true,
+      ignoreConfig: [],
+      extensions: 'js, jsx, ts, tsx',
+      ignorePattern: '**/node_modules/**',
+      parser: 'tsx',
+      runInBand: false,
+      dry: options.dryRun,
+      print: options.print,
+      verbose: options.verbose ? 2 : 0,
+      ...transformOptions,
+    })
+
+    renderSuccess({
+      headline: 'Transform complete.',
+      body: `Not changed: ${res.nochange}
+Skipped: ${res.skip}
+Errors: ${res.error}
+Success: ${res.ok}
+Time elapsed: ${res.timeElapsed}s`,
+    })
+  } catch (error) {
+    throw new AbortError(`Failed to execute transform ${transform}`)
+  }
+}
+
+interface CodeshiftConfig {
+  description?: string
+  transforms?: {[key: string]: string}
+  presets?: {[key: string]: string}
+}
+
+interface TransformInfo {
+  extensions: string[]
+  options: {
+    [key: string]: TransformInfoOption
+  }
+}
+
+interface TransformInfoOption {
+  name: string
+  type: 'string' | 'boolean'
+  description: string
+  required?: boolean
+}
+
+export async function fetchPackageConfig(npmPackageString: string, packageManager: PluginManager) {
+  let config: CodeshiftConfig | undefined
+
+  const tasks = [
+    {
+      title: `Downloading package: ${npmPackageString}`,
+      task: async () => {
+        config = await fetchPackage(npmPackageString, packageManager)
+      },
+    },
+  ]
+
+  await renderTasks(tasks)
+
+  if (!config) {
+    throw new AbortError(`Unable to locate package: ${npmPackageString}`)
+  }
+
+  return config
+}
+
+async function fetchPackage(npmPackageString: string, packageManager: PluginManager): Promise<CodeshiftConfig> {
+  async function installPackageDeps(packageName: string, version?: string): Promise<IPluginInfo> {
+    let info = await packageManager.alreadyInstalled(packageName, version)
+
+    if (info) {
+      return info
+    }
+
+    await packageManager.install(packageName, version)
+    info = packageManager.getInfo(packageName)
```

Comment:
I believe you can just do:

```suggestion
    info = await packageManager.install(packageName, version)
```

The return value of `install` [should be the same as `getInfo`](https://github.com/davideicardi/live-plugin-manager#pluginmanagerinstallname-string-version-string-promiseiplugininfo).